relayer/message_relayer.go

```diff
@@ -145,7 +145,9 @@ func (r *messageRelayer) relayMessage(unsignedMessage *avalancheWarp.UnsignedMes
 // will need to be accounted for here.
 func (r *messageRelayer) createSignedMessage() (*avalancheWarp.Message, error) {
 	r.relayer.logger.Info("Fetching aggregate signature from the source chain validators via API")
-	warpClient, err := warpBackend.NewClient(r.relayer.apiNodeURI, r.relayer.sourceBlockchainID.String())
+	// TODO: To properly support this, we should provide a dedicated Warp API endpoint in the config
```

Comment:
Can you create an issue for this if it doesn't already exist?

Reply:
Yup, this is already captured here: https://github.com/ava-labs/awm-relayer/issues/119